src/tutorials/simple-service-provider.md

```diff
@@ -724,163 +718,70 @@ Add the following block of functions and variable into the `index.ts`:
 
-            <div class="ui icon message" style="margin-left:20px;max-width: fit-content;">
-                <i class="question circle icon"></i>
-                <div class="content">
-                    <div class="header">
-                        How it works
-                    </div>
-                    <p>Once you have started your Nym Websocket client(s), this demo service providr will listen to messages that ocme through clients.</p>
-                    <p>For local testing purposes, make sure that you use the '--port' param when starting your Nym Client. Make sure that its running on a differentn port than the one thats sends messages to this application.</p>
-                    <p>Below, you can see the activity log. <b style='color: #36d481;'>Sent</b> messages will display in <b style='color: #36d481;'>green</b> while <b style='color: orange;'>received</b> messages will display in <b style='color: orange;'>orange</b>.</p>
-                </div>
-            </div>
-            
-            <h3 style="margin-left:10px">Activity Log</h3>
+* `handleResponse()` - When it comes to the sorting between types of incoming messages, its works relatively the same as the function with the same name defined in our __User Client__. The main difference is within how we are logging our data (which will be displayed in our terminal once the application is up and running).
 
-            <p style="background-color: #202124;color: #fff;">
-                <span id="output"></div>
-            </p>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js" integrity="sha512-aVKKRRi/Q/YV+4mjoKBsE4x3H+BkegoM/em46NNlCqNTmUYADjBbeNefNxYV7giUp0VxICtqdrbqU7iVaeZNXA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/semantic.min.js" integrity="sha512-Xo0Jh8MsOn72LGV8kU5LsclG7SUzJsWGhXbWcYs2MAmChkQzwiW/yTQwdJ8w6UA9C6EVG18GHb/TrYpYCjyAQw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-            <script src="index.ts"></script>
-        </body>
-    </html>
-```
+The '\x1b' orefix you see in our console.log enables the ability for us to color our 'console.log' statements. The number that you see following the '[' and preceeding 'm' is the color code that can be compared here - https://en.m.wikipedia.org/wiki/ANSI_escape_code#Colors. Its a nice and quick way of styling our terminal output.
```

Comment:
spelling and grammar fixes here - use british english spellings throughout as well please, not american english